package.json

```diff
@@ -110,6 +110,8 @@
     "@spotify/eslint-plugin": "^15.0.0",
     "@spotify/prettier-config": "^15.0.0",
     "@techdocs/cli": "workspace:*",
+    "@types/cacheable-request": "^8.3.6",
+    "@types/memjs": "^1.3.3",
```

Comment:
This fixes broken yarn build:api-docs after keyv and @keyv/memcache upgrade.

- `@types/memjs` issue is probably caused by references to older @keyv/memcache package in pinned @backstage/backend-common package. I think it will be safe to remove after getting rid of @backstage/backend-common references.

- `@types/cacheable-request` is used by `got` package that is used by some other dependencies in the repo. This way the easiest way I found to fix it.

Would appreciate if you have any suggestions on how to handle it better :)

Reply:
as long as these aren't actually needed this is an alright place to put these deps to unbreak things, since it won't affect published packages